CHANGELOG-zh-CN.md

```diff
@@ -1,5 +1,174 @@
 # OceanBase Developer Center (ODC) CHANGELOG
 
+## 4.2.3 (2023-12-21)
+
+### 功能变化
+
+数据源
+
+- 允许数据源绑定到项目
+- 支持 OceanBase Sharding MySQL 数据源
+- 支持克隆数据源
+
+导入导出
+
+- 支持原生 MySQL 数据源的导入导出
+- OceanBase 数据源的导入导出任务配置页面中不再提供 SYS 账户配置
+
+数据库对象管理
+
+- OceanBase MySQL 以及原生 MySQL 模式下支持 GIS 数据类型
+- 白屏创建或删除索引时给出高风险操作提示
+
+基于项目的管控协同
+
+- 增加 2 个内置项目角色：安全管理员，参与者；安全管理员被允许管理项目的敏感列和参与审批，参与者被允许参与审批
+- 允许用户申请项目权限
+- 禁止删除以 DBA 或 项目 OWNER 角色归属到任意项目中的用户
+
+SQL-Check
+
+- 增加了问题定位功能，允许用户将违反的安全规则和具体的 SQL 联系在一起
+
+数据库连接会话
+
+- 增加了自动重连机制，避免长时间不使用场景下会话销毁导致的报错及易用性问题
+
+分区计划
+
+- 支持定时调度
+
+SQL 执行
+
+- SQL 片段最大支持 65535 大小的内容
+- 支持 Ctrl+Enter 快捷键执行当前语句
+
+堡垒机集成
+
+- 支持 SQL Check
+
+DLM
+
+- 支持日志查看
+- 新增三个任务参数配置：查询超时时间、分片大小、分片策略
+- 优化了 MYSQL 5.6 的性能表现
+- 优化了 OceanBase 数据清理的性能表现
+
+### 缺陷修复
+
+数据源
+
+- 用户离开项目且不属于任何项目和角色时依然可以创建数据源
+- OceanBase MySQL 以及原生 MySQL 模式下数据库会话中"执行时间"栏目为 0
+- OceanBase Oracle 模式下通过会话变量管理功能进行时间输出格式的修改在 SQL 执行窗口不生效
+- OceanBase Oracle 模式下无法连接小写的 schema
+- 无法连接 percona 分支构建的 MySQL 数据源
+
+SQL 执行
+
+- SQL 执行过程中抛出错误没有国际化
+- 无法在团队空间下执行带有 dblink 的 SQL
+- 安全规则允许的前提下，无法在团队空间中执行 desc 语句
+- OceanBase Oracle 模式下执行 SELECT ... ORDER BY 1 样式的语句时 ORDER BY 会失效
+- 禁用"SQL 窗口允许执行的 SQL 类型"规则不生效
+
+结果集导出
+
+- 任务没有日志打印
+- 以 excel 格式导出后没有数据
+
+PL 对象
+
+- 交互式创建函数过程中，无法通过下拉菜单定义 sys_refcursor 类型的返回值
+- OceanBase MySQL 模式下 PL 参数值没有对单引号转义
+
+DLM
+
+- 数据库连接池过小导致任务执行失败
+
+分区计划
+
+- 在 OceanBase 1.4.79 版本的 MySQL 模式下创建任务失败
+
+SQL-Check
+
+- 无法识别`alter table xxx drop index`语句为 DROP INDEX 语句
+
+外部审批集成
+
+- 无法识别索引集合中的数据的表达式
+- 外部系统返回的 xml 形式的数据在反序列化时会丢失原始 xml 的根 tag
+
+数据脱敏
+
+- 当扫描到重复列时会导致敏感列添加失败
+
+基于项目的管控协同
+
+- 用户被赋予"个人空间"权限后必须重新登陆才能生效
+- 同步大量数据库或 schema 到项目时发生事务超时
+- 无法以项目维度过滤工单
+- 项目 OWNER 可以将项目中所有 DBA 角色的用户全部移除
+- 项目参与者可以创建数据库并将其添加到项目中
```

Comment:
May delete this as the project participant is a new feature and this issue is introduced in 4.2.3

Reply:
done